recipes/elfx86exts/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "elfx86exts" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "a536105c5ad7d392e6d75b5d0c4ffb443cf92dad9ae4a07da6aa8e164b529b18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://crates.io/api/v1/crates/{{ name }}/{{ version }}/download
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - rust
+  run:
+
```

Comment:
This `run` can be dropped. Unless there was something that was suppose to be here...